db/migrate/20210501025934_remove_contents_and_lookup_from_and_add_marked_for_deletion_to_downloads.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveContentsAndLookupFromAndAddMarkedForDeletionToDownloads < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :downloads, :lookup
+    remove_column :downloads, :contents
```

Comment:
These need a type in order for the migration to be reversible

Reply:
Added string and binary types to the remove_column and I rebased